TYPO3.Media/Classes/TYPO3/Media/Controller/AssetController.php

```diff
@@ -397,10 +397,6 @@ public function createTagAction($label)
         $existingTag = $this->tagRepository->findByLabel($label);
```

Comment:
`$assetCollection->addTag($existingTag)` will only accept `TYPO3\Media\Domain\Model\Tag` so passing the query result caused the exception in the first place